sentinel_s3/converter.py

```diff
@@ -36,15 +36,38 @@ def epsg_code(geojson):
     return None
 
 
+def test_wrap_coordinates(coords, origin, wgs84):
+    """ Test whether coordinates wrap around the antimeridian in wgs84 """
+    lon_under_minus_170 = False
+    lon_over_plus_170 = False
+    if isinstance(coords, list) or isinstance(coords, tuple):
+        try:
+            if isinstance(coords[0], list) or isinstance(coords[0], tuple):
+                return [convert_coordinates(list(c), origin, wgs84) for c in coords]
+            elif isinstance(coords[0], float):
+                c = list(transform(origin, wgs84, *coords))
+                if c[0] < -170:
+                    lon_under_minus_170 = True
+                elif c[0] > 170:
+                    lon_over_plus_170 = True
+                return c
+
+        except IndexError:
+            pass
+
+    return lon_under_minus_170 and lon_over_plus_170
+
+
 def convert_coordinates(coords, origin, wgs84):
     """ Convert coordinates from one crs to another """
+    wrapped = test_wrap_coordinates(coords, origin, wgs84)
```

Comment:
This gets call repeatedly because this is a recursive function. does it need to be called repeatedly?